src/components/SDKsPage/MainSection/index.tsx

```diff
@@ -7,27 +7,34 @@ import Link from 'src/components/Link';
 export enum Tab {
   CHANNELS = 'channels',
   SPACES = 'spaces',
+  CHAT = 'chat',
 }
 
 const MainSection = ({ tab }: { tab: Tab }) => {
-  const activeTab = tab !== Tab.SPACES ? Tab.CHANNELS : Tab.SPACES;
+  const activeTab = tab === Tab.CHAT ? Tab.CHAT : tab === Tab.SPACES ? Tab.SPACES : Tab.CHANNELS;
```

Comment:
If the goal here is to "set activeTab to tab if it's in the enum, otherwise channels" then we can simplify this nested ternary statement by examining the enum as a whole i.e.

```ts
const activeTab = Object.values(Tab).includes(tab as Tab) ? tab : Tab.CHANNELS
```